generatedocs/data/config/scheduler/examples.md

```diff
@@ -2,12 +2,12 @@
 
 If `reconnectsettings.reconnect` is enabled, the following is attempted:
 
-* Re-connect the WebSocket.
-* Get the currently opened app in the re-attached engine session.
-* Re-subscribe to the same object as before the disconnection.
-* Restart the action that was ongoing when the disconnection occurred.
-* If successful, the action during which the re-connect happened is logged as a successful action with `action` and `label` changed to `Reconnect(action)` and `Reconnect(label)`.
-* Log an info row with info type `WebsocketReconnect` and with a semicolon-separated `details` section as follows: "success=`X`;attempts=`Y`;TimeSpent=`Z`"
+1. Re-connect the WebSocket.
+2. Get the currently opened app in the re-attached engine session.
+3. Re-subscribe to the same object as before the disconnection.
+4. If successful, the action during which the re-connect happened is logged as a successful action with `action` and `label` changed to `Reconnect(action)` and `Reconnect(label)`.
+5. Restart the action that was ongoing when the disconnection occurred, unless it's a `thinktime` action which will not be restarted.
```

Comment:
One (final?) suggestion:
"5. Restart the action that was executed when the disconnection occurred (unless it is a `thinktime` action, which will not be restarted)."